source/documentation/runbooks.html.md.erb

```diff
@@ -22,7 +22,7 @@ Documentation supporting day to day processes in the Operations Engineering team
 * [Create a Sentry Internal Integration](runbooks/services/sentry-internal-integrations.html)
 * [AWS Credentials Remediation Process](runbooks/services/aws-credentials.html)
 * [OS Data Hub API Key Management](runbooks/services/os-places-api-key-management.html)
-* [Add user to Docker](runbooks/services/add-docker-user.html)
+* [Add user to Docker(Manual Process)](runbooks/services/add-docker-user.html)
```

Comment:
Do we also need to change the title of the runbook itself?

Reply:
Already done under #1486